src/Bridge/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaLengthRestriction.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Validator\Metadata\Property\Restriction;
+
+use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
+use Symfony\Component\PropertyInfo\Type;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\Length;
+
+/**
+ * Class PropertySchemaLengthRestrictions.
+ *
+ * @author Andrii Penchuk penja7@gmail.com
+ */
+class PropertySchemaLengthRestriction implements PropertySchemaRestrictionMetadataInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function create(Constraint $constraint, PropertyMetadata $propertyMetadata): array
+    {
+        $restriction = [];
+
+        switch ($propertyMetadata->getType()->getBuiltinType()) {
+            case Type::BUILTIN_TYPE_STRING:
+
+                if (isset($constraint->min)) {
+                    $restriction['minLength'] = (int) $constraint->min;
+                }
+
+                if (isset($constraint->max)) {
+                    $restriction['maxLength'] = (int) $constraint->max;
+                }
+
+                break;
+            case Type::BUILTIN_TYPE_INT:
+            case Type::BUILTIN_TYPE_FLOAT:
+                if (isset($constraint->min)) {
+                    $restriction['minimum'] = (int) $constraint->min;
+                }
+
+                if (isset($constraint->max)) {
+                    $restriction['maximum'] = (int) $constraint->max;
+                }
```

Comment:
This part looks wrong. `Length` is a string constraint. There seems to be confusion with `Range`, implemented in #4158

Reply:
That's right. Could you provide a PR to fix it?

Reply:
PR: #4177

Reply:
PS: thanks @penja for implementing this feature!

Reply:
@guilliamxavier thanks for fix